ctapipe/reco/HillasReconstructor.py

```diff
@@ -99,10 +98,14 @@ class that reconstructs the direction of an atmospheric shower
 
     """
 
-    def __init__(self, config=None, parent=None, **kwargs):
+    def __init__(self, event=None, config=None, parent=None, **kwargs):
```

Comment:
This class should not store the event in `__init__`. It should be created once and then called for each event but the event should not be state of the class.

Reply:
like CameraCalibrator?

Reply:
yes exactly